annif/annif.py

```diff
@@ -54,9 +55,24 @@ def listprojects():
 
     REST equivalent: GET /projects/
     """
-    projs = es.search(index=INDEX_NAME)['hits']
-    print(projs)
-    pass
+
+    doc = {
+            'size': 1000,
+            'query': {
+                'match_all': {}
+                }
+            }
+    results = es.search(index=annif.config['INDEX_NAME'], doc_type='project',
+            body=doc)
+    projects = [x['_source'] for x in results['hits']['hits']]
+    for p in projects:
+        print(p)
+
+    print(projects)
```

Comment:
This is redundant after the previous for loop

Reply:
Oops, true. I tried different ways to present the results when configuring tests. This I'll remove this.